src/epd2in13_v2/mod.rs

```diff
@@ -0,0 +1,558 @@
+        self.background_color = background_color;
+    }
+
+    fn background_color(&self) -> &Color {
+        &self.background_color
+    }
+
+    fn width(&self) -> u32 {
+        WIDTH
+    }
+
+    fn height(&self) -> u32 {
+        HEIGHT
+    }
+
+    fn set_lut(
+        &mut self,
+        spi: &mut SPI,
+        refresh_rate: Option<RefreshLUT>,
+    ) -> Result<(), SPI::Error> {
+        let buffer = match refresh_rate {
+            Some(RefreshLUT::FULL) | None => &LUT_FULL_UPDATE,
+            Some(RefreshLUT::QUICK) => &LUT_PARTIAL_UPDATE,
+        };
+
+        self.cmd_with_data(spi, Command::WRITE_LUT_REGISTER, buffer)
+    }
+
+    fn is_busy(&self) -> bool {
+        self.interface.is_busy(IS_BUSY_LOW)
+    }
+}
+
+impl<SPI, CS, BUSY, DC, RST> EPD2in13<SPI, CS, BUSY, DC, RST>
+where
+    SPI: Write<u8>,
+    CS: OutputPin,
+    BUSY: InputPin,
+    DC: OutputPin,
+    RST: OutputPin,
+{
+    /// When using partial refresh, the controller uses the provided buffer for
+    /// comparison with new buffer.
+    pub fn set_partial_base_buffer(
+        &mut self,
+        spi: &mut SPI,
+        buffer: &[u8],
+    ) -> Result<(), SPI::Error> {
+        assert!((WIDTH * HEIGHT / 8) as usize == buffer.len());
+        self.set_ram_area(spi, 0, 0, WIDTH - 1, HEIGHT - 1)?;
+        self.set_ram_address_counters(spi, 0, 0)?;
+
+        self.cmd_with_data(spi, Command::WRITE_RAM_RED, buffer)?;
+        Ok(())
+    }
+
+    /// Selects which sleep mode will be used when triggering the deep sleep.
+    pub fn set_deep_sleep_mode(&mut self, mode: DeepSleepMode) {
+        self.sleep_mode = mode;
+    }
+
+    /// Sets the refresh mode. When changing mode, the screen will be
+    /// re-initialized accordingly.
+    pub fn set_refresh<DELAY: DelayMs<u8>>(
+        &mut self,
+        spi: &mut SPI,
+        delay: &mut DELAY,
+        refresh: RefreshLUT,
+    ) -> Result<(), SPI::Error> {
+        if self.refresh != refresh {
+            self.refresh = refresh;
+            self.init(spi, delay)?;
+        }
+        Ok(())
+    }
+
+    // fn set_gate_scan_start_position(
+    //     &mut self,
+    //     spi: &mut SPI,
+    //     start: u16,
+    // ) -> Result<(), SPI::Error> {
+    //     assert!(start <= 295);
+    //     self.cmd_with_data(
+    //         spi,
+    //         Command::GATE_SCAN_START_POSITION,
+    //         &[(start & 0xFF) as u8, ((start >> 8) & 0x1) as u8],
+    //     )
+    // }
+
+    fn set_border_waveform(
+        &mut self,
+        spi: &mut SPI,
+        borderwaveform: BorderWaveForm,
+    ) -> Result<(), SPI::Error> {
+        self.cmd_with_data(
+            spi,
+            Command::BORDER_WAVEFORM_CONTROL,
+            &[borderwaveform.to_u8()],
+        )
+    }
+
+    fn set_vcom_register(&mut self, spi: &mut SPI, vcom: VCOM) -> Result<(), SPI::Error> {
+        self.cmd_with_data(spi, Command::WRITE_VCOM_REGISTER, &[vcom.0])
+    }
+
+    fn set_gate_driving_voltage(
```

Comment:
Some short description for these helper functions would be nice, since they are different from the normal/old control functions.